react/features/base/logging/middleware.js

```diff
@@ -141,6 +141,19 @@ function _initLogging({ dispatch, getState }, loggingConfig, isTestingEnabled) {
         const _logCollector
             = new Logger.LogCollector(new JitsiMeetLogStorage(getState));
 
+        const { apiLogLevels } = getState()['features/base/config'];
+
+        if (apiLogLevels && apiLogLevels instanceof Array) {
+            apiLogLevels.forEach(logLevel => {
+                const original = _logCollector[logLevel];
```

Comment:
Rather than a collector, doesn't this functionality fit more in a transport? All logs stored in a collector will end up in a transport, and the idea of transport is to pipe the output elsewhere. For example the mobile one put the logs in the system log.

Can you please explore the idea of adding an extra transport for this?

Reply:
switched to transport